lib/lf/uscalfe/fe_space_lagrange_o1.h

```diff
@@ -10,18 +10,18 @@
 #ifndef __7a41e223dd0e4176af0371c2b57d2b67
 #define __7a41e223dd0e4176af0371c2b57d2b67
 
-#include "scalar_uniform_fe_space.h"
+#include "uniform_scalar_fe_space.h"
 
 namespace lf::uscalfe {
 /**
  * @brief Linear Lagrangian Finite Element space
  *
- * Just a specialization of ScalarUniformFESpace based on
+ * Just a specialization of UniformScalarFESpace based on
  * FeLagrangeO1Tria, FeLagrangeO1Quad.
  *
  */
 template <typename SCALAR>
-class FeSpaceLagrangeO1 : public ScalarUniformFESpace<SCALAR> {
+class FeSpaceLagrangeO1 : public UniformScalarFESpace<SCALAR> {
```

Comment:
I have updated the lecture notes accordingly. Please be conservative about changing names of classes and functions!
Changes OK.

Reply:
thats what I was afraid of and it's also the reason I created a Pull Request. :) I will initiate a new release on hunter now.